### PR TITLE
migration: experiments

### PR DESCRIPTION
**Pull request by andrewferlitsch**
**REQUIRED:** Add a summary of your PR here, typically including why the change is needed and what was changed. Include any design alternatives for discussion purposes.

<br>
Migrating from MLops to official
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [x] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [ ] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

3. If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/community-content/CODEOWNERS) file under the `Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/official/experiments/get_started_with_vertex_experiments.ipynb [hunk 1]
```diff
@@ -0,0 +1,1581 @@
+{
```
Comment (gericdong) [inline notebooks/official/experiments/get_started_with_vertex_experiments.ipynb:1]: <p>Is there a reason this notebook is not using the latest notebook template that can significantly reduce the boilerplate? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (gericdong) [inline notebooks/official/experiments/get_started_with_vertex_experiments.ipynb:1]: <p>Typo devloping &gt; developing</p><p><a href="https://cloud.google.com/vertex-ai/docs/experiments/" target="_blank">Experiments</a>. &gt;broken link </p><p><a href="https://cloud.google.com/vertex-ai/docs/ml-metadata/introduction" target="_blank">Introduction to Vertex AI ML Metadata</a> &gt; Introduction to Vertex ML Metadata</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Comment (gericdong) [inline notebooks/official/experiments/get_started_with_vertex_experiments.ipynb:1]: <p>Typo Retreive &gt; Retrieve</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Comment (gericdong) [inline notebooks/official/experiments/get_started_with_vertex_experiments.ipynb:1]: <p>Vertex AI ML Metadata &gt; Vertex ML Metadata </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

Comment (gericdong) [inline notebooks/official/experiments/get_started_with_vertex_experiments.ipynb:1]: <p>The above works but I'd suggest to use a setup script next time to make the package assembly a lot clear.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='73'/>

Comment (andrewferlitsch) [reply notebooks/official/experiments/get_started_with_vertex_experiments.ipynb:1]: n/a

Comment (andrewferlitsch) [reply notebooks/official/experiments/get_started_with_vertex_experiments.ipynb:1]: fixed

Comment (andrewferlitsch) [reply notebooks/official/experiments/get_started_with_vertex_experiments.ipynb:1]: fixed

Comment (andrewferlitsch) [reply notebooks/official/experiments/get_started_with_vertex_experiments.ipynb:1]: fixed

Comment (andrewferlitsch) [reply notebooks/official/experiments/get_started_with_vertex_experiments.ipynb:1]: reviewed

**Review by gericdong — APPROVED** [review]
LGTM

**Pull request merged**